### DISCUSSION TITLE
使用pion进行webrtc推流，opus音频不能播放，无声音

### DISCUSSION THREAD

**Discussion by mayu531106827** [pion/webrtc#2327] [Show and tell] [answered]
使用pion进行webrtc推流，web客户端opus音频不能播放，无声音

**Comment by Sean-Der** [comment, marked as answer]
Hi @mayu531106827 

你能分享一些代码来演示发生了什么吗？

一些玩 Opus 的好例子 [play-from-disk](https://github.com/pion/webrtc/tree/master/examples/play-from-disk) [reflect](https://github.com/pion/webrtc/tree/master/examples/reflect)

**Reply by mayu531106827** [reply]
谢谢，问题已经解决了，是我的opus音频有问题。另外请教下，pion webrtc 支持h265吗？我设置mediaEngine.RegisterCodec H265并且创建了一个265的videoTrack后，在sdp信息中看不到有265的信息，现在chorme已经支持了265

**Comment by zjzhang-cn** [comment]
> 使用pion进行webrtc推流，web客户端opus音频不能播放，无声音

可能是音频没有播放器，把audiotrack 所在的流赋值给audio标签

**Reply by mayu531106827** [reply]
是我转的opus音频数据有问题
